test/flint/fq_nmod_poly-test.jl

```diff
@@ -501,6 +501,9 @@ end
   @test is_irreducible(x)
 
   @test is_irreducible(x^16+2*x^9+x^8+x^2+x+1)
+
+  @test !is_irreducible(x^0)
+  @test !is_irreducible(0*x^0)
```

Comment:
I think it would be better to add this kind of tests to the conformance test suits for polynomial rings instead. Of course we can also test it here first, no big harm (though I think at some point we should get somebody to go through the Nemo test files and weed out tests that are completely covered by the generic conformance tests)

Reply:
Sure, it can be added it to the conformance tests. But still I have the impression that the conformance tests are nice to have, but they are a bit of a black box and it is not clear to me what they guarantee to test. This here fixes a specific bug of the various Nemo methods, so I don't see why this should not be tested explicitly here.

Reply:
Yeah, to know what the conformance tests test, you need to look into them. Like so many things in AA/Nemo/Oscar that are unfortunately underdocumented :-(. At least they are relatively small, contained in a single file, and it is easy to look in there and see what is tested.

But I am not opposed to adding the tests here, I just think we should add them to the conformance tests, too. Sadly I forgot to do so when I fixed the bug here for `QQMPolyRingElem` in December, because then we'd have found & fixed the others fixed here a month earlier :-).

Anyway, I've added such tests now in https://github.com/Nemocas/AbstractAlgebra.jl/pull/1952 (and of course that PR fails for now, until this PR here is merged :-) )